minimal_azimut_angle: -135
maximal_azimut_angle: 135
azimut_angle_increment: 0.25
azimut_angle_std: 0
samples: 1081
minimal_elevation_angle: -2.5
maximal_elevation_angle: 5
elevation_angle_increment: 2.5
elevation_angle_std: 0
lasers: 4
minimal_range: 0.5
maximal_range: 64
range_std: 0
